plugin/src/main/scala/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -8,10 +8,17 @@ import nsc.plugins.{Plugin, PluginComponent}
 import scala.reflect.internal.util.NoPosition
 import scala.collection.mutable
 
-private[plugin] case class ChiselPluginArguments(val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty) {
+private[plugin] case class ChiselPluginArguments(
+  val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty,
+  var buildElementsAccessor: Boolean = true,
+  var pluginDebugBundlePattern: String = "" // For example try .*Bpip.*"
+) {
   def useBundlePluginOpt = "useBundlePlugin"
   def useBundlePluginFullOpt = s"-P:${ChiselPlugin.name}:$useBundlePluginOpt"
-
+  def buildElementAccessorOpt = "buildElementAccessor"
+  def buildElementAccessorFullOpt = s"-P:chiselplugin:$buildElementAccessorOpt"
+  def pluginDebugBundlePatternOpt = "enablePluginDebug"
+  def pluginDebugBundlePatternFullOpt = s"-P:chiselplugin:$pluginDebugBundlePatternOpt:"
```

Comment:
Can you add some comments here to describe how to use the debugpattern stuff? Is that something we want merged or is it just for debugging during development?

Reply:
I removed this from here. Added a docs/src/appendix/chisel-plugin.md to describe debug techniques